src/cargo/ops/registry/info/view.rs

```diff
@@ -169,7 +169,7 @@ fn pretty_source(source: SourceId, ctx: &GlobalContext) -> String {
         .local_path()
         .and_then(|path| pathdiff::diff_paths(path, ctx.cwd()))
     {
-        let path = std::path::Path::new(".").join(relpath);
+        let path = std::path::Path::new("../../../../..").join(relpath);
```

Comment:
This doesn't look right

Reply:
Oops. I believe it changed when I tried to drag it into the subdirectory.